bleak/backends/corebluetooth/utils.py

```diff
@@ -68,3 +74,19 @@ def to_optional_int(value: Optional[NSNumber]) -> Optional[int]:
         return None
 
     return int(value)
+
+
+_Ts = TypeVarTuple("_Ts")
+
+
+def call_soon_threadsafe(
```

Comment:
This could be useful in the WinRT backend as well since it has a similar situation with event callbacks.

So perhaps we should start a new `bleak.backends._util` module for this?